src/popup/components/TransactionList.vue

```diff
@@ -197,17 +232,30 @@ export default defineComponent({
 
     async function getLatest() {
       try {
-        await root.$store.dispatch('fetchTransactions', { limit: 10, recent: true });
+        await root.$store.dispatch('fetchTransactions', { limit: 10, recent: true, address: currentAddress.value });
       } finally {
         loading.value = false;
       }
     }
 
+    async function setMultisigTransaction(acc: IMultisigAccount | undefined) {
+      if (acc) {
+        multisigTransaction.value = {
+          ...await fetchActiveMultisigTx(acc.contractId),
+        };
+      }
+    }
+
     watch(displayMode, () => {
       checkLoadMore();
     });
 
-    onMounted(() => {
+    onMounted(async () => {
+      if (isMultisigDashboard.value) {
```

Comment:
The `loadMore` and setInterval logic should be wrapped in `else` block` as fetching the regular transactions is not needed when browsing the multisig ones.

Reply:
It shouldn't, we should fetch all of the transactions